lib/src/utils/themes/code_block_theme.dart

```diff
@@ -0,0 +1,130 @@
+import 'package:flutter/material.dart';
+
+/// The the theme applied to the code block
+class CodeBlockTheme {
+  final TextStyle root;
+  final TextStyle comment;
+  final TextStyle quote;
+  final TextStyle tag;
+  final TextStyle attribute;
+  final TextStyle keyword;
+  // ignore: non_constant_identifier_names
+  final TextStyle selector_tag;
+  final TextStyle literal;
+  final TextStyle name;
+  final TextStyle variable;
+  // ignore: non_constant_identifier_names
+  final TextStyle template_variable;
+  final TextStyle code;
+  final TextStyle string;
+  // ignore: non_constant_identifier_names
+  final TextStyle meta_string;
+  final TextStyle regexp;
+  final TextStyle link;
+  final TextStyle title;
+  final TextStyle symbol;
+  final TextStyle bullet;
+  final TextStyle number;
+  final TextStyle section;
+  final TextStyle meta;
+  final TextStyle type;
+  // ignore: non_constant_identifier_names
+  final TextStyle built_in;
+  // ignore: non_constant_identifier_names
+  final TextStyle builtin_name;
```

Comment:
You can change these variable names. Just be careful because this will affect the `_toMap()` function.